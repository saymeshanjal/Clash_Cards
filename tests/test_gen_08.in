10
4 47 2
10 20 3
10 49 3
4 45 2
10 47 1
14 6 2
2 49 1
10 14 2
16 41 2
10 48 2
