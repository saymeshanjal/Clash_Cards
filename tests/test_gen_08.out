1 2 3
35 217 0
110 142 0
1 2 3
126 126 0
36 1128 2268
1 1 0
8 42 202
353 992 11525
76 77 99
